src/outputs.c

```diff
@@ -225,6 +225,19 @@ void deinit_outputs() {
         evdev = NULL;
     }
 }
+void singletap(uint32_t timestamp_ms, imu_quat_type quat, imu_euler_type velocities, bool ipc_enabled,
+                       bool imu_calibrated, ipc_values_type *ipc_values) {
+    libevdev_uinput_write_event(uinput, EV_KEY, BTN_LEFT,1);
+    libevdev_uinput_write_event(uinput, EV_SYN, SYN_REPORT, 0);
+    libevdev_uinput_write_event(uinput, EV_KEY, BTN_LEFT,0);
+    libevdev_uinput_write_event(uinput, EV_SYN, SYN_REPORT, 0);
+}
+
+void doubletap(uint32_t timestamp_ms, imu_quat_type quat, imu_euler_type velocities, bool ipc_enabled,
+                       bool imu_calibrated, ipc_values_type *ipc_values) {
+    singletap(timestamp_ms, quat, velocities, ipc_enabled, imu_calibrated, ipc_values);
+    singletap(timestamp_ms, quat, velocities, ipc_enabled, imu_calibrated, ipc_values);
```

Comment:
I think there should be a delay here? Just to make it seem a little more human in case some apps care. Like most OS's allow you to configure expectations around double-click.